framework/tester.py

```diff
@@ -144,11 +144,22 @@ def start_all_servers(self):
             if not srv.is_running():
                 raise Exception("Can not start server %s" % id)
 
+    def wait_all_connections(self, tmt=1):
+        for id in self.__servers:
+            srv = self.__servers[id]
+            self.assertTrue(srv.wait_for_connections(timeout=tmt))
+
     def start_tempesta(self):
         self.__tempesta.start()
         if not self.__tempesta.is_running():
             raise Exception("Can not start Tempesta")
 
+    def set_nf_mark(self, mark):
+        self.__tempesta.set_mark(mark)
```

Comment:
Why do we need this function? Why can not we set mark from tests

Reply:
Corrected.